src/chainparams.cpp

```diff
@@ -443,6 +443,13 @@ class CCustomParams : public CRegTestParams {
         std::vector<unsigned char> man_bytes = ParseHex(gArgs.GetArg("-con_mandatorycoinbase", ""));
         consensus.mandatory_coinbase_destination = CScript(man_bytes.begin(), man_bytes.end()); // Blank script allows any coinbase destination
 
+        // Block signing encumberance script, default of 51 aka OP_TRUE
+        std::vector<unsigned char> sign_bytes = ParseHex(gArgs.GetArg("-signblockscript", "51"));
+        consensus.signblockscript = CScript(sign_bytes.begin(), sign_bytes.end());
+        // Default signature size is the size of dummy push, and single 72 byte DER signature
```

Comment:
I just figured a default of allowing a single DER encoded ECSDA sig was appropriate in case people want to switch to single-signer? If you think that's stupid I can change it to 0 :)

Reply:
I think this is fine. This means we don't need to change this param ever once MuSig lands :)